src/com/company/Main.java

```diff
@@ -2,7 +2,94 @@
 
 public class Main {
 
-    public static void main(String[] args) {
-	// write your code here
+    public static <Char> void main(String[] args) {
+	// Задача 1
+        System.out.println("Задача 1");
+        String firstName="Ivan";
+        String middleName="Ivanovich";
+        String lastName="Ivanov";
+        String fullName=lastName+" "+firstName+" "+middleName;
+        System.out.println("ФИО сотрудника — "+fullName);
+        System.out.println();
+
+        // Задача 2
+        System.out.println("Задача 2");
+        String fullNameUpper=fullName.toUpperCase();
+        System.out.println("Данные ФИО сотрудника для заполнения отчета — "+fullNameUpper);
+        System.out.println();
+
+        // Задача 3
+        System.out.println("Задача 3");
+        String fullNameForAdministration = fullName.replace(" ", "; ");
+        System.out.println("Данные ФИО сотрудника для административного отдела — "+fullNameForAdministration);
+        System.out.println();
+
+        // Задача 4
+        System.out.println("Задача 4");
+        fullName="Иванов Семён Семёнович";
+        String correctedFullName = fullName.replace("ё", "e");
+        System.out.println("Данные ФИО сотрудника — "+correctedFullName);
+        System.out.println();
+
+        //Задачи повышенной сложности
+        System.out.println("Задачи повышенной сложности");
+        System.out.println();
+
+        //Задача 5
+        System.out.println("Задача 5");
+        String fullName2="Ivanov Ivan Ivanovich";
```

Comment:
для лучшей читаемости следует оставлять пробелы, разделяя оператор и операнды.
Можете использовать ctrl+alt+L для автоформатирования
```suggestion
        String fullName2 = "Ivanov Ivan Ivanovich";
```